### ISSUE TITLE
gnuplot script generates error

### ISSUE THREAD

**Issue by vuki** [rgbkrk/atom-script#841]
One of the latest changes, "add gnuplot script", prevents the package from loading and also does not allow Atom to load subsequent packages. The error in console is:

```
Cannot read property 'file' of undefined TypeError: Cannot read property 'file' of undefined
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/grammars.coffee:131:7)
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/grammars.coffee:4:1)
    at Module._compile (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/code-context-builder.coffee:2:14)
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/code-context-builder.coffee:1:1)
    at Module._compile (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:50:27)
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/script.coffee:1:22)
    at Object.<anonymous> (file:///C:/Users/Greg/.atom/packages/script/lib/script.coffee:1:1)
    at Module._compile (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\package.js:720:27)
    at Package.module.exports.Package.activateConfig (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\src\package.js:220:12)
    at PackageManager.<anonymous> (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\node_modules\settings-view\lib\package-manager.js:55:14)
    at PackageManager.packageHasSettings (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\node_modules\underscore-plus\node_modules\underscore\underscore.js:658:63)
    at PackageCard.module.exports.PackageCard.hasSettings (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\node_modules\settings-view\lib\package-card.js:611:34)
    at PackageCard.module.exports.PackageCard.initialize (C:\Users\Greg\AppData\Local\atom\app-1.5.4\resources\app.asar\node_modules\settings-view\lib\package-card.js:205:17)
Uncaught (in promise) TypeError: Cannot read property 'hide' of undefined(…)
'KeyboardEvent.keyLocation' is deprecated. Please use 'KeyboardEvent.location' instead.
```

Removing gnuplot script from grammar file fixed the error.

Latest atom 1.5.4, Windows 7, program started by "atom ."

**Comment by nixel2007** [comment]
I think i have a fix.
Give me a few minutes to test

**Comment by nixel2007** [comment]
@vuki may i ask you to try this version of gnuplot support? can reproduce bug on my machine

```coffee
  gnuplot:
    "File Based":
      command: "gnuplot"
      args: (context) -> ['-p', context.filepath]
      workingDirectory: atom.workspace.getActivePaneItem()?.buffer?.file?.getParent?().getPath?()
```

**Comment by vuki** [comment]
Yes, it works now.

**Comment by nixel2007** [comment]
Fixed in 3.6.1 :+1:
